sound/soc/sof/amd/acp.c

```diff
@@ -516,6 +518,16 @@ int amd_sof_acp_probe(struct snd_sof_dev *sdev)
 		return ret;
 	}
 
+	sdev->dsp_box.offset = 0;
+	sdev->dsp_box.size = BOX_SIZE_512;
+
+	sdev->host_box.offset = sdev->dsp_box.offset + sdev->dsp_box.size;
+	sdev->host_box.size = BOX_SIZE_512;
+
+	sdev->debug_box.offset = sdev->host_box.offset + sdev->host_box.size;
+	sdev->debug_box.size = BOX_SIZE_1024;
```

Comment:
This is a bit confusing when reading the commit message and the patch.
The size of inbox, outbox and debug box remains the same (512, 512 and 1024), no?

The change I can see in this patch is that `scratch_ipc_conf` lost the out/in_box members, thus `scratch_ipc_conf` and `scratch_reg_conf` is moved up to referenced from the `debug_box.offset`

Reply:
In probe we are setting this for backward compatible.

AS we see the window information is defined in the firmware ,now we get the updated inbox and outbox sizes from the firmware there this mailboxes are set to 1024. The window info from firmware PR (https://github.com/thesofproject/sof/pull/6002)

Reply:
Done